deku-core/src/Deku/Core.purs

```diff
@@ -630,14 +410,14 @@ useRef a b f = Deku.do
 withUnsub :: Effect Unit -> PSR -> PSR
 withUnsub u (PSR psr) = PSR $ psr { unsubs = Array.snoc psr.unsubs u }
 
-useRefST :: forall a. a -> Poll a -> Hook (ST Global a)
+useRefST :: forall a. a -> Poll a -> Hook (ST.ST Global a)
 useRefST a e f = Nut $ mkEffectFn2 \psr di -> do
-  r <- liftST $ STRef.new a
+  r <- liftST $ ST.new a
   { event, push } <- liftST $ Event.create
-  u <- subscribe (Poll.sample e event) \i -> void $ liftST $ STRef.write i r
+  u <- Event.subscribe (Poll.sample e event) \i -> void $ liftST $ ST.write i r
   push identity
-  let Nut nut = f (STRef.read r)
-  runEffectFn2 nut (withUnsub u psr) di
+  let Nut nut = f ( ST.read r )
```

Comment:
Small nit - there's a lot of stuff like this that must come from your formatter. We can merge it in, but it may get swapped back if the `purs-tidy` in my IDE is run on it. If you want this formatting to be part of the repo permanently, that's fine, and in that case we should commit some formatting pre-commit hook to make sure it's standardized.

Reply:
I'm a whitespace maximalist but unfortunatly purs-tidy does not support that yet. So until I can force my will on the world I'll just let the formatter do its thing.